frame/grandpa/src/mock.rs

```diff
@@ -199,6 +199,7 @@ impl onchain::Config for Test {
 }
 
 impl pallet_staking::Config for Test {
+	const MAX_NOMINATIONS: u32 = 16;
```

Comment:
One advantage of using `Get<u32>` here instead of a const is that technically we could have some runtime storage control this item versus ONLY ever letting it be a const.

Reply:
not sure if this would be compatible with the changes I am seeing below though

Reply:
If I make this a non-const item, then the `ElectionDataProvider::MAXIMUM_NOMINATOR_PER_VOTER` should also be non-const i.e. function or `type _ = Get<_>`. 

I am fine with both, just wanted to point out why I did it so far as such, they are intertwined. I'd say we change it when the need arises, for now const should be enough.

Reply:
agreed, this is good for now. thanks!